Panels/FrameTrackViewer.js

```diff
@@ -724,6 +724,32 @@ define([
 
             };
 
+            /**
+             * Zooms to the range posStart-posEnd, with some additional space
+             * on both sides of the range, as specified with marginFraction.
+             *
+             * @param posStart Range start position (inclusive).
+             * @param posEnd Range end position (inclusive).
+             * @param marginFraction Fraction of the total range width to leave
+             *        as margin on both sides.
+             */
+            panel.zoomToRange = function(posStart, posEnd, marginFraction = 0.1) {
```

Comment:
Zooming in/out indefinitely  is not possible. I'm not sure how its enforced in general, but it means 
 `rangeSize`  cant be too small or too big.  Will that be corrected by marginFraction or is it something else?

Reply:
If you zoom in to a single position (let's say range 100, 101), then that position will be at the center of your screen, zoomed in at the maximum possible zoom level (`_maxZoomFactor` I think).

If you zoom out to a range larger than what is currently shown in the track viewer, then it will zoom out as much as possible, so that the entire range is shown.

`posEnd` should be larger than `posStart` of course. There is no check for that, but I think that is pretty much in line with the rest of the code.

`marginFraction` will add some extra space around the defined position range. This is currently used in the CNV viewer, for zooming in to a CNV with a bit of space around it. This function was already present in the `GenomeBrowser`. I only moved it here to be able to reuse it for the split viewers.